common/processor/processor/src/main/java/io/helidon/common/processor/TypeFactory.java

```diff
@@ -87,9 +87,10 @@ public static Optional<TypeName> createTypeName(TypeMirror typeMirror) {
         case TYPEVAR -> {
             return Optional.of(createFromGenericDeclaration(typeMirror.toString()));
         }
-        case WILDCARD -> {
+        case WILDCARD, ERROR -> {
```

Comment:
shouldn't we have a unit test for these returns?

Reply:
Can we get a `TypeMirror` instance in a unit test? These methods did not have unit tests before, so I guess you investigated this?